### DISCUSSION TITLE
Please guide how to place (website + database) and (media files) on 2 different servers?

### DISCUSSION THREAD

**Discussion by tippytea** [mediacms-io/mediacms#514] [Q&A] [answered]
As an example.
I will apply the mediacms.tv domain to make a video website.

I would like to install the website and database (www.mediacms.tv) using server A, which has a small hard drive but good quality and fast bandwidth.

I want to install the media files uploaded by users on another server dedicated to storage and transcoding, use mediafiles.mediacms.tv to access these media files, and the media file called for the content of the main site is https://mediafiles.mediacms.tv/xxxxx/xxx.m3u8

In addition to this main problem, there are 4 other minor problems that I would like you to guide me on.

1, after installing this source code by default, I found that it will automatically notify ffmpeg transcoding many resolutions, from 240-1080 each, which will obviously fill up the hard disk space very quickly, if I only want it to transcode 2 resolutions: 480P and 720P, how do I need to set it?

2. How can I set it to automatically add a watermark to each uploaded video during the transcoding process? I saw that someone submitted a suggestion about watermark in 2021 https://github.com/mediacms-io/mediacms/issues/66, is the watermark feature developed now?

thanks for you work.

**Comment by mgogoulos** [comment, marked as answer]
for 1 go to https://your_portal.org/admin/files/encodeprofile/ and make sure enabled is only what you want. Do that before you start uploading content!

**Reply by mgogoulos** [reply]
2. there's no such thing implemented

**Reply by tippytea** [reply]
> 
thank you for answer.

and this : how to place (website + database) and (media files) on 2 different servers?

**Reply by mgogoulos** [reply]
This is not  supported functionality for the vanilla installation, so you need to find your own solution I guess with nginx and proxying from the front server (A) to server (B)
